spec.html

```diff
@@ -40029,36 +40015,36 @@ <h1>
           1. Return the Element Type value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _O_.[[TypedArrayName]].
         </emu-alg>
       </emu-clause>
-    </emu-clause>
 
-    <emu-clause id="sec-comparetypedarrayelements" type="abstract operation">
-      <h1>
-        CompareTypedArrayElements (
-          _x_: unknown,
-          _y_: unknown,
-          _comparefn_: a function object or *undefined*,
-        ): either a normal completion containing a Number or an abrupt completion
-      </h1>
-      <dl class="header">
-      </dl>
-      <emu-alg>
-        1. Assert: _x_ is a Number and _y_ is a Number, or _x_ is a BigInt and _y_ is a BigInt.
-        1. If _comparefn_ is not *undefined*, then
-          1. Let _v_ be ? ToNumber(? Call(_comparefn_, *undefined*, « _x_, _y_ »)).
-          1. If _v_ is *NaN*, return *+0*<sub>𝔽</sub>.
-          1. Return _v_.
-        1. If _x_ and _y_ are both *NaN*, return *+0*<sub>𝔽</sub>.
-        1. If _x_ is *NaN*, return *1*<sub>𝔽</sub>.
-        1. If _y_ is *NaN*, return *-1*<sub>𝔽</sub>.
-        1. If _x_ &lt; _y_, return *-1*<sub>𝔽</sub>.
-        1. If _x_ > _y_, return *1*<sub>𝔽</sub>.
-        1. If _x_ is *-0*<sub>𝔽</sub> and _y_ is *+0*<sub>𝔽</sub>, return *-1*<sub>𝔽</sub>.
-        1. If _x_ is *+0*<sub>𝔽</sub> and _y_ is *-0*<sub>𝔽</sub>, return *1*<sub>𝔽</sub>.
-        1. Return *+0*<sub>𝔽</sub>.
-      </emu-alg>
-      <emu-note>
-        This performs a numeric comparison rather than the string comparison used in <emu-xref href="#sec-comparearrayelements"></emu-xref>.
-      </emu-note>
+      <emu-clause id="sec-comparetypedarrayelements" type="abstract operation">
+        <h1>
+          CompareTypedArrayElements (
+            _x_: a Number or a BigInt,
+            _y_: a Number or a BigInt,
```

Comment:
this should accompany changing the assertion in step 1 to `Assert: Type(_x_) is Type(_y_)`

Reply:
I'll merge this into the existing PR as is for now. Feel free to add the comment back on the primary PR.